src/chrome/content/rules/148apps.com.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	Nonfunctional hosts in *.148apps.com:
+	    beta.images.148apps.com
+	Redirect to "www.148apps.com":
```

Comment:
You should add them anyway because a lot of resources are loaded over those domains.